src/test/java/software/amazon/cloudformation/resource/ValidatorTest.java

```diff
@@ -39,6 +39,7 @@
 public class ValidatorTest {
     private static final String RESOURCE_DEFINITION_SCHEMA_PATH = "/schema/provider.definition.schema.v1.json";
     private static final String TEST_SCHEMA_PATH = "/test-schema.json";
+    private static final String SPECIAL_SCHEMA_PATH = "/special-schema.json";
```

Comment:
can we use a different name? special isn't super descriptive

Reply:
may be DRIFT_METADAT_SCHEMA_PATH?